packages/fxa-payments-server/src/components/LoadingOverlay.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { LoadingOverlay } from './LoadingOverlay';
+
+storiesOf('components/LoadingOverlay', module)
+  .add('isLoading = true', () => (
+    <LoadingOverlay isLoading={true} />
+  ))
+  .add('isLoading = false', () => (
```

Comment:
This shows up as... a blank page? Maybe that makes sense, but also seems weird...?

Reply:
The loader shouldn't appear at all when not loading, so technically that's what this shows.